### PR TITLE
Rogelio

### PR DESCRIPTION
**Pull request by rogelio1603**
Check the new commit about redux in the project

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,12 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <Provider store={store}></Provider>;
```
Comment (kikiondo117) [inline src/App.js]: The Wrapper needs to be in the index.js (main node) file

File: src/components/NuevoProducto.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const NuevoProducto = () => {
```
Comment (kikiondo117) [inline src/components/NuevoProducto.jsx]: English the name file and component 
._.)/

File: src/components/Productos.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Productos = () => {
+  return <h1>En productos</h1>;
```
Comment (kikiondo117) [inline src/components/Productos.jsx]: The same above bro

File: src/containers/app/App.js [hunk 4]
```diff
@@ -1,7 +1,15 @@
-import React from 'react';
+import React, { Fragment } from 'react';
+import Producto from '../../components/NuevoProducto';
+import Productos from '../../components/Productos';
 
 function App() {
-  return <h1>Desde App</h1>;
+  return (
+    <Fragment>
```
Comment (kikiondo117) [inline src/containers/app/App.js:7]: Why are you using Fragment ?

File: src/index.js [hunk 5]
```diff
@@ -1,12 +1,18 @@
...
 import reportWebVitals from './reportWebVitals';
 
+//Redux
+import { Provider } from 'react-redux';
+import store from './store/store';
+
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
```
Comment (kikiondo117) [inline src/index.js:13]: This is the correct please, so why do you have two providers ?

File: src/reducers/index.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux';
+import productosReducer from './productosReducer';
+
+export default combineReducers({
+  productos: productosReducer,
+});
```
Comment (kikiondo117) [inline src/reducers/index.js]: Change the name's file as rootReducer

File: src/reducers/productosReducer.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  error: null,
+  loading: false,
+};
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    default:
+      return state;
+  }
+}
```
Comment (kikiondo117) [inline src/reducers/productosReducer.js:14]: A suggestion is install redux-actions library or immer to handler the reducer state and mutation

File: src/store/store.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import { createStore, applyMiddleware } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+
+import reducer from '../reducers';
```
Comment (kikiondo117) [inline src/store/store.js]: Change the name to rootReducer

**Review by kikiondo117 — CHANGES_REQUESTED** [review]

File: src/containers/app/App.js (hunk 4 above)
Comment (rogelio1603) [reply src/containers/app/App.js:7]: I can only return 1 component, for that reason I used Fragment to wrap Product and Products component.

**Comment by rogelio1603** [comment]
The changes has been complete

**Review by kikiondo117 — APPROVED** [review]

**Pull request merged**